tests/images/e2e/Dockerfile

```diff
@@ -1,9 +1,10 @@
-FROM alpine:3.10
+FROM debian:buster-slim
```

Comment:
installing aws cli2 into alpine requires some hack (need glibc compatible and special LD_LIBRARY_PATH to load .so library files in its directory)
this is a small image which has better support for most software